src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,155 @@
+package duke.task;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private List<Task> list;
+    private static final String starline = "**************************************************************************\n";
```

Comment:
Should constant names be all uppercase with underscore as word separators? (See Java coding standard -> Naming)

Reply:
Edited accordingly. Thank you! :)